swap_meet/item.py

```diff
@@ -1,2 +1,28 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        
+        if id is None:
+            unique_id = uuid.uuid4()
+            self.id = unique_id.int
```

Comment:
👍 The `Item` constructor properly sets its attributes and allows for default parameters. Similar to needing new lists in the Vendor constructor, here, we need to run a function for the default case, which means that we need a sentinel to mark that no value was supplied so that we can then run the id generation logic within the constructor, ensuring that every item gets its own id.